### DISCUSSION TITLE
github page cors error please help

### DISCUSSION THREAD

**Discussion by East-cloud-of-forest** [community/community#37317] [Other Feature Feedback, Questions, & Ideas]
I am making a page that receives xml from open api without a server.
In devserver, I solved the cors using a proxy, but I don't know how to build it on the github page.
I know that my repositories are public and access-control-allow-origin will be * when github pages are public. But I keep getting the cors error.

![image](https://user-images.githubusercontent.com/92777293/197987698-ec0a7d9c-7ddb-47d7-8e14-69a5a0dfcbd5.png)
![image](https://user-images.githubusercontent.com/92777293/197987796-02d7748c-1ac6-4522-aa26-013b1121cb7f.png)
![image](https://user-images.githubusercontent.com/92777293/197987879-773dfe72-f1ca-4ea1-b467-30ddfdbc7fe1.png)
![image](https://user-images.githubusercontent.com/92777293/197987948-55afe1f6-018a-4771-a7a9-a3dc13e780f0.png)
![image](https://user-images.githubusercontent.com/92777293/197987977-5d05bc1c-f776-4276-b656-da437604cdeb.png)
![image](https://user-images.githubusercontent.com/92777293/197988025-cfeda604-8a29-48bb-b0cb-74cd1d94d14f.png)
![image](https://user-images.githubusercontent.com/92777293/197988265-3bcd2a07-d1ee-4833-be90-97f039a0f31b.png)

Open API only supports xml format.
please help

**Comment by airtower-luna** [comment]
The problem is that the `openapi.11st.co.kr` server doesn't send an [`Access-Control-Allow-Origin` header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Allow-Origin), which means your browser isn't allowed to provide the response data to your site. That is a security mechanism, see [Cross-Origin Resource Sharing (CORS)](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) for details.

You cannot solve that inside your site, it's something the `openapi.11st.co.kr` admins would have to change (if they want to). Alternatively you'd need your own server that retrieves the data and makes it available to your site.

**Reply by East-cloud-of-forest** [reply]
thank you so much
